### PR TITLE
Navigation style improvements

### PR DESCRIPTION
**Pull request by Krystian22FrontEnd**
Changed some styling to match desired layout and responsiveness of the page.

### CODE REVIEW

File: src/common/Navigation/styled.js [hunk 1]
```diff
@@ -156,29 +157,3 @@ export const NavigationInput = styled.div`
     margin-bottom: 16px;
   }
 `;
```
Comment (tomecky1) [inline src/common/Navigation/styled.js:159]: You have kept ot all the active classname for the Navlink? Looks pretty good for me.

File: src/App.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ function App() {
   return (
     <Container>
       <NaviRoutes />
-      <MovieDetails />
+      {/* <MovieDetails /> */}
```
Comment (tomecky1) [inline src/App.js:12]: I see you have uncommented Movie details, does it work ?

Comment (tomecky1) [reply src/App.js:12]: I meant otherwise - commented, sorry about that mistake :)

**Review by tomecky1 — APPROVED** [review]
I think it's ready for merge.

File: src/App.js (hunk 2 above)
Comment (Krystian22FrontEnd) [reply src/App.js:12]: it's ok

File: src/common/Navigation/styled.js (hunk 1 above)
Comment (Krystian22FrontEnd) [reply src/common/Navigation/styled.js:159]: Yeah they are pretty usefull

**Pull request merged**